src/components/UserInfoForm/UserInfoForm.tsx

```diff
@@ -0,0 +1,336 @@
+                      onClose={(
+                        event: React.SyntheticEvent<unknown>,
+                        reason?: string
+                      ) => {
+                        if (reason !== "backdropClick") {
+                          setOpen(false);
+                        }
+                      }}
+                    >
+                      <DialogTitle>Select Country</DialogTitle>
+                      <DialogContent>
+                        <Box
+                          component="form"
+                          sx={{ display: "flex", flexWrap: "wrap" }}
+                        >
+                          <FormControl
+                            sx={{ m: 1, minWidth: 230, minHeight: 140 }}
+                          >
+                            <MenuItem value=""></MenuItem>
+                            {Object.keys(countries).map((country, index) => (
+                              <Button
+                                key={index}
+                                className={formClasses.CountryBtn}
+                                onClick={() => FlagsChanger(countries[country])}
+                              >
+                                <img
+                                  src={countries[country].icon}
+                                  alt={countries[country].value}
+                                  className={formClasses.CountryImg2}
+                                />
+                                {countries[country].value}
+                              </Button>
+                            ))}
+                          </FormControl>
+                        </Box>
+                      </DialogContent>
+                    </Dialog>
+                  </InputAdornment>
+                ),
+              }}
+            />
+            <TextField
+              error={!!errors.email}
+              fullWidth
+              label="Email"
+              className={formClasses.TxtBottom}
+              {...register("email", {
+                required: true,
+                pattern: /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/,
+              })}
+              helperText={!!errors.email && <p>Invalid Email!</p>}
+            />
+            <TextField
+              error={!!errors.password}
+              fullWidth
+              type="password"
+              label="Password"
+              id="filled-adornment-password"
+              className={formClasses.TxtBottom}
+              {...register("password", {
+                required: true,
+                minLength: 8,
+                pattern: /^(?:(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).*)+$/,
+              })}
+              helperText={
+                !!errors.password && (
+                  <p>
+                    Oops! You need a password longer than 8 characters with
+                    numbers and letters.
+                  </p>
+                )
+              }
+            />
+            <input
```

Comment:
use button here

Reply:
input field is replaced is with MUI button